From 1da6ececc6dbebc1f10a8f517edad0f56d45d256 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 13 Dec 2017 20:19:20 +0900
Subject: [PATCH 022/909] arm64: renesas: Enable GPIOLIB

commit 17e84ad7896f13ec9252b8b629f7b9929e4e4b5c from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

The R-Car GPIO driver does not enable when Renesas SoC's ARCH configs
(ARCH_RENESAS, ARCH_R8A7795, ARCH_R8A7796 and ARCH_R8A77965) only.

This patch enables the R-Car GPIO driver by selecting GPIOLIB for
the Renesas platform.

Reported-by: Alexandru Gheorghe <Alexandru_Gheorghe@mentor.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/Kconfig.platforms | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 21a715ad8222..7b3ff54360ed 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -171,6 +171,7 @@ config ARCH_SYNQUACER
 config ARCH_RENESAS
 	bool "Renesas SoC Platforms"
 	select ARCH_SHMOBILE
+	select GPIOLIB
 	select PINCTRL
 	select PM
 	select PM_GENERIC_DOMAINS
-- 
2.17.1

